gradle-plugins/buildSrc/src/main/kotlin/printAllAndroidxReplacements.kt

```diff
@@ -0,0 +1,66 @@
+import io.ktor.client.*
+import io.ktor.client.request.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.filter
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.runBlocking
+
+private const val libsRepo = "https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/"
+private const val version = "0.3.0-build154"
```

Comment:
Please mark it with __LATEST_COMPOSE_RELEASE_VERSION__

Reply:
I think it is better to read this version from gradle.properties